packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -116,14 +123,18 @@ module.exports = {
       endsWith: "string/ends-with",
       fromCodePoint: "string/from-code-point",
       includes: "string/includes",
-      padLeft: "string/pad-left",
-      padRight: "string/pad-right",
+      padLeft: "string/pad-start", // deprecated
+      padRight: "string/pad-end", // deprecated
```

Comment:
Since these moved, it'll mean if people have babel-runtime pinned but the transform as a semver range, their build would break, and same the other way around I think. Maybe we should leave these as-is and special-case them in the build script instead so pad-left loads pad-start?